### ISSUE TITLE
Swipe / keyboard navigation

### ISSUE THREAD

**Issue by goosechaser** [booncon/slippry#28]
There are some many sliders that you gotta have something amazing to stand out. Lack of simple features such as swipe left/right and keyboard navigation is a big no-no. I opened your site, tried to swipe, failed, closed and moved on. That's how most of those who are looking for a plugin will behave. And, as a user, you will get frustrated and not interact with the slider if that functionality is missing.

Just an opinion :)
